BusinessTampereTrafficMonitoring/traffic_lights/signal_group.py

```diff
@@ -12,6 +22,15 @@ def __init__(self, device: str, name: str, timestamp: str, status_code: str):
             self.t_red_start = timestamp
 
     def __create_event(self, timestamp):
+        """
+
+        To create the traffic light cycle event # Parameter: # argument1: timestamp (TimeStamp)
+        # Returns: # Device
+        id (String), device name(string), red_light start time(TIMESTAMP), green_light start_time(TIMESTAMP),
```

Comment:
Try to be consistent with the empty lines. Why is this all on the same line? The types are wrong.